diffkemp/diffkemp.py

```diff
@@ -117,6 +120,8 @@ def generate(args):
     # Build sources for symbols from the list into LLVM IR
     with open(args.functions_list, "r") as fun_list_file:
         for line in fun_list_file.readlines():
+            # enable comments inside functions list
+            line = line[:line.find('#')]
```

Comment:
Use `"` for string literals instead of `'` (many occurrences in other places, too).